packages/dx-react-chart/src/types/templates.series.types.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  Scales, BuildAnimatedStyleGetterFn, TransformedPoint,
+} from '@devexpress/dx-chart-core';
+import { PureComputed } from '@devexpress/dx-core';
+
+type Path = PureComputed<[TransformedPoint[]], string>;
+
+interface InternalPathProps {
+  /** @internal */
+  index: number;
+  /** @internal */
+  state?: string;
+  /** @internal */
+  style?: any;
+  /** @internal */
+  scales: Scales;
```

Comment:
`scales`, `state`, `getAnimatedStyle` should not be declared in several interfaces - there should be one (like the code that makes these fields in *series-helper*.

Something like
```js
interface <Connected>XXX {
  index: number;
  state?: string;
  scales: Scales;
  getAnimatedStyle: GetAnimatedStyleFn;
}

interface <Connected>Series extends <Connected>XXX {
  pointComponent: ...
}

interface <Connected>Point extends <Connected>XXX {
  seriesIndex: number;
}
```